src/bootstrap-table.js

```diff
@@ -724,7 +724,7 @@ class BootstrapTable {
         $(currentTarget).val(text)
       }
 
-      if (this.searchText === text) {
+      if (this.searchText === text && text.length > 0) {
```

Comment:
ref #4399, @UtechtDustin help to review this line.

Reply:
i will try to help the next days, currently im ill. sorry !